pkg/gotohelm/transpiler.go

```diff
@@ -114,6 +126,20 @@ func (t *Transpiler) Transpile() *Chart {
 	return &chart
 }
 
+func addPackagePrefixIfNotExist(f *ast.File, name string) {
```

Comment:
Package prefixes, as a concept, already exist. They're called "namespaces" (`Transpiler.namespaces`). Let's re-use that concept here?